### ISSUE TITLE
Strange <nil> token. Should be None?

### ISSUE THREAD

**Issue by zurk** [bblfsh/python-driver#99]
I found it in my visualization example:
https://gist.github.com/zurk/d314d67d9aac8843d3776c82cd738b40
```
8  ||             ┣ BINARY, THIS, EXPRESSION                                     
8  |var1|         ┃ ┣ LEFT, IDENTIFIER, EXPRESSION                               
7  ||             ┃ ┃ ┣ INCOMPLETE                                               
7  |\n|           ┃ ┃ ┗ ┗ DOCUMENTATION                                          
8  |<nil>|        ┃ ┗ BYTE, NUMBER, EXPRESSION, RIGHT 
```
If you take a look at the code, you will see that actually <nil> corresponds to None token.
Also, I doubt that None should have a `NUMBER` role, maybe `NULL`?

**Comment by zurk** [comment]
ok, If you just parse `None` via Babelfish, you will have this tree:
```
  ||      FILE                        
1 ||      ┣ EXPRESSION                
1 |<nil>| ┗ ┗ BYTE, NUMBER, EXPRESSION
```
and I expected to get something like:
```
  ||      FILE                        
1 ||      ┣ EXPRESSION                
1 |None|  ┗ ┗ NULL, EXPRESSION
```
May be something else, idk.

**Comment by zurk** [comment]
Also `THIS` role is strange in the example.

**Comment by juanjux** [comment]
Was because of the desync between the SDK numeric roles, fixed in the latest version that should be incorporated in the python-driver 1.0.3 image.

It's `Literal, Null, Expression` in the last version.

**Comment by zurk** [comment]
Yes, but the main question of issue is about `<nil>` token. It should be `None` I think.

**Comment by juanjux** [comment]
Oh I skipped that one! I'll be fixed in PR #112 which includes misc annotation fixes (mostly from your reports). It'll probably be merged in version 1.0.4 of the driver, but this version could take some time since we're in the middle of a refactor of several components.

**Comment by juanjux** [comment]
Fixed by #112 and #113 when merged.
